api/tests/integration/infrastructure/repositories/authentication-method-repository_test.js

```diff
@@ -94,4 +94,110 @@ describe('Integration | Repository | AuthenticationMethod', () => {
       });
     });
   });
+
+  describe('#findOneByUserIdAndIdentityProvider', () => {
+
+    it('should return the AuthenticationMethods associated to a user for a given identity provider', async () => {
```

Comment:
```suggestion
    it('should return the AuthenticationMethod associated to a user for a given identity provider', async () => {
```

Puisqu'on ne cherche qu'à en prendre un

Reply:
Bien vu